test/test/components/previews/inline_component_preview.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class InlineComponentPreview < ViewComponent::Preview
+  def default
+    render(InlineComponent.new)
+  end
+
+  def inside_form
+    render_with_template
```

Comment:
It'd be _really_ great if we could avoid having to call `render_with_template` here. Rails controllers allow for implicit rendering, and I think we could do the same.

Reply:
Gotcha

Reply:
https://github.com/github/view_component/pull/353/commits/2edcaf622a5212ca5788b8c41c16689380b75cad
Would something like this work?

Reply:
I know we initially talked about not having not even the method definition in the preview object, but I feel that would introduce some unnecesary complexity to the codebase. I know it's kinda strange to have body-less methods, but since this lives in the test land,I would encourage for explicitness rather than the other way around. 

It's just a thinking, if you feel that it would be better to avoid the method definition anyways I can give it a try.